src/igvfd/schemas/biosample.json

```diff
@@ -251,18 +251,18 @@
                 "decline"
             ]
         },
-        "biosample_ontology": {
-            "title": "Biosample ontology term",
-            "description": "Ontology term of the biosample.",
+        "biosample_term": {
+            "title": "Biosample term",
+            "description": "Schema for an ontological term identifying a biosample.",
```

Comment:
This is an incorrect description. biosample_term is a link to an object representing an ontological term, not schema ...

Reply:
done